src/Data/Certificate.php

```diff
@@ -39,6 +49,7 @@ public function __construct($privateKey, $csr, $certificate)
         $this->privateKey = $privateKey;
         $this->csr = $csr;
         $this->certificate = $certificate;
+        list($this->certificateNoChain, $this->intermediateCertificate) = Helper::splitCertificate($certificate);
```

Comment:
since we support php 7+ we should use a shorthand here `[...]` instead of `list(...)`

Reply:
This will make it php >= 7.1 rather than php >= 7.0, is that ok?

Reply:
I have made the change assuming your intent was not to bump up the php version requirement to stay php 7+ compatible (leaving the list command in place). I have implemented your regex suggestion, and also added an exception throw if it cannot be parsed.

Reply:
Awesome, I just did some simple naming changes, and merged it. Thanks for the idea and working on this!